src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -120,6 +126,7 @@ function unhandledError(connection: signalR.HubConnection, err: Error, logger: I
   //
   // Trying to call methods on the connection after its been closed will throw.
   if (connection) {
+    renderingFailed = true;
```

Comment:
@rynowak Your idea for displaying an error UI would be extremely valuable here. @javiercn and I were discussing this earlier.

As it stands, if there's a rendering error, the app will disconnect and stop responding, but the user won't be given any visual indication of what's going on. The UI will just freeze.

I think that's OK for right now, but makes me think we should prioritise including the error UI.